src/main/java/com/kdt_y_be_toy_project2/domain/trip/controller/TripController.java

```diff
@@ -0,0 +1,53 @@
+package com.kdt_y_be_toy_project2.domain.trip.controller;
+
+import com.kdt_y_be_toy_project2.domain.trip.dto.*;
+import com.kdt_y_be_toy_project2.domain.trip.service.TripService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/trips")
+@RequiredArgsConstructor
+public class TripController {
+    private final TripService tripService;
+
+    @GetMapping
+    public ResponseEntity<List<GetTripResponse>> getAllTrips() {
+        return ResponseEntity.status(HttpStatus.OK)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(tripService.getAllTrips());
+    }
+
+    @GetMapping("/{trip_id}")
+    public ResponseEntity<GetTripResponse> getTripById(
+            @PathVariable(name = "trip_id") final Long tripId
+    ) {
+        return ResponseEntity.status(HttpStatus.OK)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(tripService.getTripById(tripId));
+    }
+
+    @PostMapping
+    public ResponseEntity<CreateTripResponse> createTrip(
+            @RequestBody final CreateTripRequest request
```

Comment:
```suggestion
            @Valid @RequestBody final CreateTripRequest request
```
`@Valid` 를 추가한다면 Spring Level 에서 validation 을 감지할 수 있습니다.